echopype/tests/test_echodata.py

```diff
@@ -133,3 +137,33 @@ def _check_path(zarr_path):
             and converted_zarr.endswith(".nc")
         ):
             assert isinstance(e, ValueError) is True
+
+
+@pytest.mark.parametrize(
+    ("filepath", "sonar_model", "xml_path"),
+    [
+        (ek60_path / "ncei-wcsd" / "Summer2017-D20170615-T190214.raw", "EK60", None),
+        (ek80_path / "D20170912-T234910.raw", "EK80", None),
+        (azfp_path / "ooi" / "17032923.01A", "AZFP", azfp_path / "ooi" / "17032922.XML"),
+        (ad2cp_path / "raw" / "090" / "rawtest.090.00001.ad2cp", "AD2CP", None)
+    ]
+)
+def test_compute_range(filepath, sonar_model, xml_path):
+    ed = echopype.open_raw(filepath, sonar_model, xml_path)
+    sound_speed = 343
+    if sonar_model == "EK60":
+        range = ed.compute_range(sound_speed, ek_waveform_mode="CW")
+    elif sonar_model == "EK80":
+        range = ed.compute_range(sound_speed, ek_waveform_mode="CW")
```

Comment:
@imranmaj : Is there a test for `ek_waveform_mode="BB"` or is there a reason why that is not included here?